monkey/common/types.py

```diff
@@ -1,8 +1,19 @@
+from ipaddress import IPv4Address
 from uuid import UUID
 
-from pydantic import PositiveInt
+from pydantic import PositiveInt, conint
 from typing_extensions import TypeAlias
 
+from common.base_models import InfectionMonkeyBaseModel
+
 AgentID: TypeAlias = UUID
 HardwareID: TypeAlias = PositiveInt
 MachineID: TypeAlias = PositiveInt
+
+
+class SocketAddress(InfectionMonkeyBaseModel):
+    ip: IPv4Address
+    port: conint(ge=1, le=65535)  # type: ignore
```

Comment:
Be more specific about what mypy errors we're ignoring. Otherwise we could potentially ignore an issue that we should have addressed.

```suggestion
    port: conint(ge=1, le=65535)  # type: ignore[valid-type]
```